### PR TITLE
[Vm-repair] Vm repair bug fixes

### PR DESCRIPTION
**Pull request by haagha**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Vm-repair

**Comment by zhoxing-ms** [comment]
Could you please write the description of PR change into the `HISTORY.rst`? And if you want to release a new extension version for this PR, please also upgrade the version defined in the `setup.py` file.

File: src/vm-repair/azext_vm_repair/_params.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def load_arguments(self, _):
         c.argument('enable_nested', help='enable nested hyperv.')
         c.argument('associate_public_ip', help='Option to create repair vm with public ip')
         c.argument('distro', help='Option to create repair vm from a specific linux distro (rhel7|rhel8|suse12|ubuntu20|centos7|oracle7)')
+        c.argument('yes', help='Option to skip associate_public_ip prompt and confirm yes to it in no Tty mode')
```
Comment (zhoxing-ms) [inline src/vm-repair/azext_vm_repair/_params.py:35]:
```suggestion
        c.argument('yes', help='Option to skip prompt for associating public ip and confirm yes to it in no Tty mode')
```

**Comment by haagha** [comment]
Added description for History and bumped up the version. Also fixed the parameter help. Thanks!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**